### ISSUE TITLE
[Idea] Run Multiple Accounts Simultaneously 

### ISSUE THREAD

**Issue by aangel031** [ahmadudin/electron-instaPy-GUI#26]
It would be a great feature if you can run the bot with multiple accounts at the same time

**Comment by ahmadudin** [comment]
Yeah, great idea. I already have it on my list but currently i am so busy. I'll try to implement it when i have spare time.

**Comment by hamsgrow4be** [comment]
Hi Ahmadudin, 

If you can get to run multiple accounts we will pay you for that. Please take our contact for more information: grow4be@gmail.com

Waiting for you answer.
